src/sdk/keygen.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  TfheClientKey,
+  TfheConfigBuilder,
+  TfheCompactPublicKey,
+} from "./fhe/tfhe";
+
+export const createTfheKeypair = () => {
+  // const block_params = new ShortintParameters(
+  //   ShortintParametersName.PARAM_MESSAGE_2_CARRY_2_COMPACT_PK_PBS_KS,
+  // );
+  const config = TfheConfigBuilder.default()
+    //..(block_params)
+    .build();
+  const clientKey = TfheClientKey.generate(config);
+  let publicKey = TfheCompactPublicKey.new(clientKey);
+  publicKey = TfheCompactPublicKey.deserialize(publicKey.serialize());
```

Comment:
I'm not really sure, I moved the file from the tests folder into the src folder without changes other than import paths.
I moved it so that I could export `createTfhePublicKey` and use it in tests in hardhat-plugin.